tests/security/stig/oscap_security_guide_setup.pm

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test "stig" hardening in the "scap-security-guide" works: setup environment
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93886, poo#104943
+
+use base 'stigtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # Install packages
+    zypper_call('in openscap-utils scap-security-guide', timeout => 180);
+
+    # Record the pkgs' version for reference
+    my $out = script_output("zypper se -s openscap-utils scap-security-guide");
+    record_info("Pkg_ver", "open scap security guide packages' version: $out");
+
+    # Set ds file
+    $self->set_ds_file();
+
+    # Check the ds file information for reference
+    my $f_ssg_sle_ds = $stigtest::f_ssg_sle_ds;
+    $out = script_output("oscap info $stigtest::f_ssg_sle_ds");
```

Comment:
should use the parameter `$f_ssg_sle_ds`, it's been defined above.

Reply:
Good pointing. I will revise.

Reply:
done